WorkloadManagementSystem/Utilities/PilotCStoJSONSynchronizer.py

```diff
@@ -128,7 +129,8 @@ def _getCSDict(self):
     :rtype: S_OK, S_ERROR, value is pilotDict
     """
 
-    pilotDict = {'Setups': {}, 'CEs': {}, 'GenericPilotDNs': []}
+    pilotDict = {'timestamp': str(datetime.datetime.utcnow()),
```

Comment:
How about using ISO 8601 rather than the Python string representation?
```suggestion
    pilotDict = {'timestamp': datetime.datetime.utcnow().isoformat(),
```